mlflow/pyfunc/__init__.py

```diff
@@ -1691,6 +1852,19 @@ def spark_udf(
     NOTE: Inputs of type ``pyspark.sql.types.DateType`` are not supported on earlier versions of
     Spark (2.4 and below).
 
+    NOTE: If using Databricks Connect to connect to remote Databricks cluster,
+       the Databricks cluster must use runtime version greater than 16, or use runtime version
```

Comment:
`greater than 16` means `> 16`. Is that correct?

Reply:
You are right. it should be >=16, updated.